commons-httpclient/src/main/java/org/apache/commons/httpclient/HttpMethodDirector.java

```diff
@@ -150,6 +150,7 @@ public void executeMethod(final HttpMethod method) throws IOException, HttpExcep
         
                 // get a connection, if we need one
                 if (this.conn == null) {
+                    connectionManager.closeIdleConnections(60000);
```

Comment:
I suggest closeIdleConnections value should be configurable better than hardcore. 
WDYT?

Reply:
I think so too.

Then how can I get the value from config file, for example axis2_blocking_client.xml ?

or.. how about set the SO_TIMEOUT value like below.
```
connectionManager.closeIdleConnections(connectionManager.getParams().getSoTimeout());
```